lib/streamlit/config.py

```diff
@@ -858,22 +858,15 @@ def _server_address() -> Optional[str]:
 )
 
 
-@_create_option(
-    "browser.serverPort",
-    visibility="hidden",
-    deprecated=True,
-    deprecation_text="browser.serverPort has been deprecated. It will be removed in a future version.",
-    expiration_date="2024-04-01",
-    type_=int,
-)
+@_create_option("browser.serverPort", type_=int)
 def _browser_server_port() -> int:
     """Port where users should point their browsers in order to connect to the
```

Comment:
nit: maybe we can also point out here that this is only useful for advanced usecases and that `server.port` is the correct option to actually change the port of the app.

Reply:
Done.